packages/xo-web/src/xo-app/home/host-item.js

```diff
@@ -221,6 +221,15 @@ export default class HostItem extends Component {
                 <li>{_('longerCustomCertficate')}</li>
                 <li>{_('longerDefaultCertificate')}</li>
               </ul>
+              <Tooltip>
+                <a
+                  href={`https://docs.xcp-ng.org/releases/release-8-3/#host-certificate-key-too-small-prevents-upgrade`}
+                  target='_blank'
+                  rel='noreferrer'
+                >
+                  <Icon icon='info' /> {_('pubKeyTooShortLink')}
+                </a>
+              </Tooltip>
```

Comment:
Why `Tooltip`?

Reply:
Indeed i used a Tooltip for my first try but after discussing with Samuel we went with the link and I forgot to delete it, my bad